packages/infra/config/env-config.ts

```diff
@@ -95,8 +95,9 @@ export type EnvConfig = {
     CW_GATEWAY_AUTHORIZATION_CLIENT_ID: string;
     CW_GATEWAY_AUTHORIZATION_CLIENT_SECRET: string;
   };
-  iheGateway?: {
-    certArn: string;
+  ihe?: {
```

Comment:
Wasn't iheGateway the correct name for this?

Reply:
I modified the name since now it is more than just the gateway. It is the ihe lambdas and the ihe gateway and will have to point to mirth in some capacity soon too

Reply:
Gotcha (re: lambdas)

AFAIK IHEGateway = Mirth, no?

Reply:
Meant to say Api gateway sorry. I think the architecture has an api gateway sitting in front of mirth.

Reply:
@jonahkaye the lambda responses are part of the gateway solution, so we can just leave the name to be ihe gateway